pkg/pillar/cmd/client/parseuuid.go

```diff
@@ -117,3 +144,19 @@ func generateConfigRequest() ([]byte, error) {
 	}
 	return b, nil
 }
+
+func generateUUIDRequest() ([]byte, error) {
+	deviceCertPem, err := ioutil.ReadFile(types.DeviceCertName)
```

Comment:
Given that we only do this using the V2 API, which includes the full device cert in the container, why do we even need the same device cert in the payload?

Reply:
yes, I agree. Accordingly I'veupdated UuidRequest to be empty, just to force authContainer.